assets/swg-button.css

```diff
@@ -257,3 +257,17 @@
 .swg-button-dark:lang(zh-tw):after {
   background-image: url("i18n/b-zh-tw-dk.svg");
 }
+
```

Comment:
Please move it up before all the language re-definitions.

Reply:
Moved up.